05-foundational-skills_1.Rmd

```diff
@@ -145,4 +143,45 @@ dataedu::course_data
 
 ## Steps for working through new and unfamiliar content
 
-Need to add
\ No newline at end of file
+One skill that all great educators possess is the ability to ask questions. 
+Asking the right questions at the right time of the learners in your classroom can facilitate understanding, uncover misconceptions, and indicate whether or not learners have mastered the material.
+
+However, when you’re learning on your own you have to simultaneously fill the roles of both learner and educator, and not only know both how and when to ask yourself questions, but also answer your questions, evaluate your answers, and redirect your learning path as you progress.
+
+This section is intended to give you a series of steps you can use as you encounter new and unfamiliar content both in reading this book as well as in your broader data science learning endeavors.
+For this section we'll use the example of encountering a function for the first time, but you can use these steps with any new piece of information that you encounter!
+We'll cover functions in greater depth in subsequent chapters, but for now all you need to know about a function is that it's a reusable piece of code that allows us to consistently repeat a programming task. 
+
+**Activate prior knowledge**
+You've been reading through a tutorial and have come across the `coalesce()` function. 
+Take a moment to think through the following questions:
+
+* What does the word "coalesce" mean?
+* Have you ever seen the `coalesce()` function before? If so, in what context?
+
+**Look for context clues**
+
+* Read a couple of lines of code both above and below where the `coalesce()` function appears - are there any clues as to what this function might do?
```

Comment:
wondering what you think about adding 2-3 lines of code here - with the middle line including the `coalesce()` function? Perhaps these could be inspired from some use of `coalesce()` in the wild?